server/lib/accounts.js

```diff
@@ -310,3 +310,17 @@ Accounts.validateNewUser(function(user) {
 
 	return true;
 });
+
+export const MAX_RESUME_LOGIN_TOKENS = 50;
+
+Accounts.onLogin(async ({ user }) => {
+	if (user && user.services && user.services.resume && user.services.resume.loginTokens) {
```

Comment:
change to a "negative condition" and return to avoid nesting and keep code slim.